src/50_Samples_%26_Templates/Product.html

```diff
@@ -121,10 +137,34 @@
     <body>
     <%>product/header.html%>
 
-        <!-- This button adds the item to the shopping cart. Pressing this button should lead to a the shopping cart page on your normal website and the cart should be prefilled with the current item. -->
+        <!-- This dropdown adds the item to the shopping cart. Pressing the `ADD TO CART` button should lead to a lighbox which will allow to choose whether to continue to the shopping cart page or to continue with your shopping. The shopping cart page will be on your normal website and the cart should be prefilled with the current item. -->
         <div id="product-header">
             <h2>Apple</h2>
-            <a href="/shopping_cart/" class="button button-primary call-to-action">Buy</a>
+            <div class="call-to-action">
+                <select>
+                  <option>1</option>
+                  <option>2</option>
+                  <option>3</option>
+                </select>
+                <form method="post"
+                  action-xhr="<%host%>/samples_templates/product_listing/preview/add_to_cart"
+                  target="_blank"
+                  on="submit-success:success-lightbox;submit-error:error-lightbox">
+                  <input type="submit" class="button button-primary" value="ADD TO CART">
+                </form>
+            </div>
+            <amp-lightbox id="success-lightbox" class="lightbox" layout="nodisplay">
+                <div class="lightbox-content">
+                    <a href="/shopping_cart/" class="button button-primary call-to-action">Go to Cart</a>
```

Comment:
Do you know if it's possible to use amp-mustache inside an amp-lightbox? It doesn't seem to work for me.